meson.build

```diff
@@ -100,6 +100,7 @@ add_project_arguments([
     '-D__USE_MINGW_ANSI_STDIO',
     '-D_POSIX',
     '-D_WIN32_WINNT=0x0A00',
+    '-D_NO_SYS_GUID_OPERATOR_EQ_',
```

Comment:
What's this for?

Reply:
Windows defines global == operators which conflict with these:
https://github.com/opendcdiag/opendcdiag/blob/c73dc66bd62ca4ea62c2a1f3fabc8862dd9e7d03/framework/sandstone_tests.h#L88-L89

Reply:
Got it. It deserves a comment, though.